src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers.Tests/AZC0002Tests.cs

```diff
@@ -38,20 +40,22 @@ await Verifier.CreateAnalyzer(code)
         public async Task AZC0002ProducedForMethodsWithWrongNameCancellationToken()
         {
             const string code = @"
+using Azure;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace RandomNamespace
 {
     public class SomeClient
     {
-        public virtual Task {|AZC0002:GetAsync|}(CancellationToken cancellation = default)
+        public virtual Response {|AZC0002:GetAsync|}(CancellationToken cancellation = default)
```

Comment:
Would it make sense for tests of convenience methods to use `Response<Model>` instead of just response, to better simulate a real client? Should we include tests for `Pageable<Model>` and `Operation<Model>` as well?

Reply:
It is just unit test and we are not testing the response. What's the value we could gain to change to `Response<Model>`? 

> Should we include tests for Pageable<Model> and Operation<Model>

We have a lot of cases in AZC0018Tests.